Tests/SonarQube.TeamBuild.Integration.Tests/SonarQube.TeamBuild.Integration.Tests.csproj

```diff
@@ -50,7 +50,7 @@
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Setup.Configuration.Interop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.VisualStudio.Setup.Configuration.Interop.1.11.2273\lib\net35\Microsoft.VisualStudio.Setup.Configuration.Interop.dll</HintPath>
-      <EmbedInteropTypes>True</EmbedInteropTypes>
+      <EmbedInteropTypes>False</EmbedInteropTypes>
```

Comment:
I don't fully understand this one. When setting it to `true`, two tests (Initialize_CanGetGetExeToolPathFromSetupConfiguration and Initialize_CanGetGetExeToolPathFromSetupConfigurationForBuildAgent) fail with missing assembly error:

System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.Setup.Configuration.Interop, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

Reply:
It's probably not worth spending much time on as it's only in a test project.

FYI it looks like _SonarQube.TeamBuild.Integration.csproj_ is only product project that has _EmbedInteropTypes_ set to true. It references version 1.14.114 of the _XXX.Configuration.Interop.dll_ package. It's probably worth quick test to see if referencing the same package in this project fixes the issue.

Reply:
This didn't work :(